Source/WebInspectorUI/UserInterface/Views/ConsoleSession.js

```diff
@@ -46,7 +46,7 @@ WI.ConsoleSession = class ConsoleSession extends WI.Object
             headerText = WI.UIString("Page reloaded at %s");
             break;
         case WI.ConsoleSession.NewSessionReason.PageNavigated:
-            headerText = WI.UIString("Page navigated at %s");
+            headerText = WI.UIString("Navigated to %s at %s");
```

Comment:
personally i think there's still value in having "page" somewhere in here so id recommend `"Page navigated to %s at %s"`